src/graphql/syncDataSilos.ts

```diff
@@ -168,7 +173,7 @@ export async function fetchAllSubDataPoints(
       };
     }>(SUB_DATA_POINTS, {
       first: PAGE_SIZE,
-      dataPoints: [dataPointId],
+      dataPointIds: [dataPointId],
```

Comment:
unrelated bug that I found during testing

Reply:
oh ya...... noticed this the other day trying to demo, left a note, needed to make a ticket. nice find!